example/lib/meeting/meeting_store.dart

```diff
@@ -686,6 +686,12 @@ class MeetingStore extends ChangeNotifier
             audioTrack: track as HMSAudioTrack));
         notifyListeners();
       }
+      ///In order to avoid errors because of 
+      ///track updates ordering for audio and video
+      ///adding the method call here.
+      if (spotlightMetadata == track.trackId) {
```

Comment:
This looks like a potential code smell. Ideally, changing the order of when the Spotlight is set should not cause the issue.